app/packages/embeddings/src/Color.tsx

```diff
@@ -15,3 +15,306 @@ export class Color {
     return `rgb(${this.r * 255}, ${this.g * 255}, ${this.b * 255})`;
   }
 }
+
+export const cssColorNames = {
```

Comment:
Why is this needed? Could use a comment

Reply:
Color can be valid css names now, so added this constant to convert the name to hex value.